src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="#"
+    method="post"
+    data-qa="big"
+    class="search big-search"
+    >
+      <label class="container big-container">
+        <img src="./images/Search.svg"
+       alt="icon"
+       class="image big-images"
+      >
+        <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        class="input big-input"
+      >
+      </label>
+    </form>
+    <form
+    action="#"
+    method="post"
+    data-qa="small"
+    class="search search-small"
+  >
+      <label class="container small-cont">
+        <img src="./images/Search.svg"
+          alt="small-icon"
+          class="image small-image"
+        >
+        <input
+       type="text"
+       placeholder="Try &quot;Los Angeles&quot;"
+       class="input small-input"
+      >
```

Comment:
- Each level of nesting requires 2 indents
- closing tag should have the same indentation as an opening one
```suggestion
           type="text"
           placeholder="Try &quot;Los Angeles&quot;"
           class="input small-input"
        >
```